examples/build_bcd_payload.py

```diff
@@ -4,6 +4,9 @@
 that can be used in the pymodbus library. Below is a
 simple binary coded decimal builder and decoder.
 """
+
```

Comment:
Please remove, an example is not a library.

Reply:
Removed